src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -165,32 +166,33 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
   @Test
   fun `complete draft adjudication`() {
     prisonApiMockServer.stubPostAdjudication()
-    bankHolidayApiMockServer.stubGetBankHolidays()
+    val intTestData = IntTestData(webTestClient, jwtAuthHelper, bankHolidayApiMockServer, prisonApiMockServer)
 
-    val draftAdjudicationResponse = dataAPiHelpers().startNewAdjudication(dateTimeOfIncident = DATE_TIME_OF_INCIDENT)
-    dataAPiHelpers().addIncidentStatement(draftAdjudicationResponse.draftAdjudication.id, "test statement")
+    val firstDraftUserHeaders = setHeaders(username = IntTestData.DEFAULT_ADJUDICATION.createdByUserId)
+    val draftAdjudicationResponse = intTestData.startNewAdjudication(IntTestData.DEFAULT_ADJUDICATION, firstDraftUserHeaders)
+    intTestData.addIncidentStatement(draftAdjudicationResponse, IntTestData.DEFAULT_ADJUDICATION, firstDraftUserHeaders)
 
     webTestClient.post()
       .uri("/draft-adjudications/${draftAdjudicationResponse.draftAdjudication.id}/complete-draft-adjudication")
       .headers(setHeaders())
       .exchange()
       .expectStatus().isCreated
       .expectBody()
-      .jsonPath("$.adjudicationNumber").isEqualTo(1524242)
-      .jsonPath("$.prisonerNumber").isEqualTo("A12345")
-      .jsonPath("$.bookingId").isEqualTo("1")
-      .jsonPath("$.dateTimeReportExpires").isEqualTo("2010-11-15T10:00:00")
-      .jsonPath("$.incidentStatement.statement").isEqualTo("new statement")
-      .jsonPath("$.incidentDetails.dateTimeOfIncident").isEqualTo("2010-11-12T10:00:00")
-      .jsonPath("$.incidentDetails.handoverDeadline").isEqualTo("2010-11-15T10:00:00")
-      .jsonPath("$.incidentDetails.locationId").isEqualTo(721850)
+      .jsonPath("$.adjudicationNumber").isEqualTo(IntTestData.DEFAULT_ADJUDICATION.adjudicationNumber)
+      .jsonPath("$.prisonerNumber").isEqualTo(IntTestData.DEFAULT_ADJUDICATION.prisonerNumber)
+      .jsonPath("$.bookingId").isEqualTo(1) // From PrisonAPIMockServer.stubPostAdjudication
```

Comment:
We really ought to address this - the bookingId should be part of the test data

Reply:
I'm being a bit slow I'm not sure I understand this comment

Reply:
The code should read something like `isEqualTo(IntTestData.DEFAULT_ADJUDICATION.bookingId)` instead of being hard-coded - and that means adding the `bookingId` to the test data sets